test/integration/test/captions.test.js

```diff
@@ -0,0 +1,28 @@
+const clientHelper = require('../src/client-helper');
+const configHelper = require('../src/config-helper');
+
+test('Can get srt captions', async (done) => {
+    const client = clientHelper.getClient(configHelper.getApiKey());
+    const jobList = await client.getListOfJobs();
+    const jobId = getTranscribedJobId(jobList);
+    expect(jobId).toBeDefined();
```

Comment:
This test would fail in an environment/account that has not previously submitted a job.

Reply:
You are correct. The SDK is using the same user as the other integration tests and since those run so often we should be ok for now. I will file a follow up ticket to address this in every test that expects a completed job is present.